mobile-app/app/screens/AppNavigator/screens/Loans/screens/BorrowLoanTokenScreen.tsx

```diff
@@ -72,6 +75,8 @@ export function BorrowLoanTokenScreen ({
     new BigNumber(loanToken.activePrice?.active?.amount ?? 0),
     interestPerBlock
   )
+  const { network } = useNetworkContext()
+  const blocksPerDay = network === EnvironmentNetwork.MainNet || network === EnvironmentNetwork.TestNet ? 2880 : 144
```

Comment:
I think this one is used somewhere else right? Maybe we can replace with hooks?

Reply:
updated